ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -48,10 +53,20 @@
         ViewTreeObserver.OnGlobalLayoutListener {
     private static final int FLING_SCALE_FACTOR = 20;
 
+    private static final int DELAY_FIRST_BOUNCE_ANIMATION = 1000;
+    private static final int DELAY_BETWEEN_BOUNCE_ANIMATION = 10000;
+    private static final int BOUNCE_ANIMATION_START_DELAY = 500;
+    private static final long BOUNCE_ANIMATION_DURATION_FIRST_BOUNCE = 300;
```

Comment:
haha, I just re-read the name I gave this. it's a poor name because it can be mis-interrupted. I'm going to adjust this name before I merge.